ts/src/index.ts

```diff
@@ -72,6 +73,12 @@ function initConfigLocal(config: Config): ProfilerConfig {
   let envSetConfig: Config = {};
   const val = process.env.GCLOUD_PROFILER_CONFIG;
   if (val) {
+    try {
+      const envSetConfigBuf = fs.readFileSync(val);
+      envSetConfig = JSON.parse(envSetConfigBuf.toString());
```

Comment:
It is good to keep your `try` block focussed. I think you are intending only to catch the the `JSON.parse()` error here. Reduce the `try` block to just that.

Reply:
Removed try/catch block.
I think it may make sense to just let the error be thrown.